bottlecap/src/config/env.rs

```diff
@@ -38,6 +38,7 @@ pub struct Config {
     pub logs_config_logs_dd_url: String,
     pub serverless_flush_strategy: FlushStrategy,
     pub enhanced_metrics: bool,
+    pub proc_enhanced_metrics: bool,
```

Comment:
should we include anything `lambda|serverless` related?

Reply:
Hmmm, why? Today `enhanced metrics` as a concept only apply to lambda/serverless

Reply:
idk, just trying to be cautious about adding new env vars

Reply:
okay I can add lambda

Reply:
just a nit, i trust your gut 🫡